src/internal_events/http_client.rs

```diff
@@ -9,14 +12,30 @@ pub struct AboutToSendHTTPRequest<'a, T> {
     pub request: &'a Request<T>,
 }
 
+fn remove_sensitive(headers: &HeaderMap<HeaderValue>) -> HeaderMap<HeaderValue> {
+    let mut headers = headers.clone();
+    let redacted = HeaderValue::from_static("<redacted>");
```

Comment:
We can use [HeaderValue::set_sensitive](https://docs.rs/http/0.2.1/http/header/struct.HeaderValue.html#method.set_sensitive)

Reply:
I agree, let's use `HeaderValue::set_sensitive` here!

Reply:
Thanks!